### ISSUE TITLE
Does not support absolute external URLs for stylesheets by default?

### ISSUE THREAD

**Issue by aripollak** [Mange/roadie#106]
According to the README, this gem doesn't touch external CSS links. But if I try to use one without a `data-roadie-ignore`, it fails under roadie 3.0.4

```ruby
irb(main):004:0> Roadie::Document.new('<link rel="stylesheet" type="text/css" href="http://foo.com/foo.css">').transform
Roadie::CssNotFound: Could not find stylesheet "http://foo.com/foo.css"
Used provider:
ProviderList: [
	#<Roadie::FilesystemProvider:0x007f5f7d5b3748>
]
	from /home/ari/.rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/roadie-3.0.4/lib/roadie/asset_provider.rb:8:in `find_stylesheet!'
```

**Comment by Mange** [comment]
That's not supposed to happen. I'll fix it.

**Comment by Mange** [comment]
It should be fixed now on `master`. I'll release a new version later, as this is a pretty small bug fix. Is that okay with you?

**Comment by aripollak** [comment]
:+1: Thanks for the quick fix!

**Comment by Mange** [comment]
I just released version 3.0.5 now with this fix in it. Thank you for reporting and for your patience! :heart:

**Comment by dimroc** [comment]
This breaks Rails' emails that use a CDN. See #107.
